### DISCUSSION TITLE
skynet 适合写web服务器吗？ 还是配合nodejs等传统web框架做互通？

### DISCUSSION THREAD

**Discussion by lsx1994** [cloudwu/skynet#1976] [General]
需求不多，就是登录/获取服务器列表 等游戏相关功能。
不知道skynet作为web服效率如何，与Openresty对比呢？

**Comment by terry8210** [comment]
建议用其他 例如go

**Comment by findix** [comment]
我自己在Skynet里实现了一个，支持服务端模板渲染和路由之类的，实际在线项目用起来是没啥问题

**Comment by hanxi** [comment]
试试 https://github.com/hanxi/wlua ? 我用它写过一个web后台 https://github.com/hanxi/skynet-admin

**Comment by Zwlin98** [comment]
这类需求全都可以用外挂 [Moon](https://github.com/Zwlin98/moon) 解决，用 Go 写 Web，有各种现成的框架可以用。
